src/main/java/mate/academy/spring/controller/InjectController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.Role;
+import mate.academy.spring.service.AuthenticationService;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/inject")
+public class InjectController {
+    private RoleService roleService;
+    private AuthenticationService authenticationService;
+    private UserService userService;
+
+    @Autowired
```

Comment:
we have to use  @Autowired annotation everywhere above constructors or nowhere(for code consistency)